sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -1,10 +1,10 @@
 # Azure Form Recognizer client library for Python
 
-Azure Form Recognizer is a cloud service that uses machine learning to analyze text and structured data from your documents. It includes the following main features:
+Azure Document Intelligence (previously Form Recognizer) is a cloud service that uses machine learning to analyze text and structured data from your documents. It includes the following main features:
```

Comment:
I left only this mention to the name change to help folks understand...although it could be a note instead. Let me know